src/components/Genres/Genres.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./Genres.module.scss";
+
+const Genres: React.FC<{
+  genres: string[];
+  selectedGenre: string;
+  onSelect: (genre: string) => void;
+}> = ({ genres, selectedGenre, onSelect }) => {
+  return (
+    <div className={styles["genres"]}>
+      {genres.map((genre) => (
+        <button
+          key={genre}
+          onClick={() => onSelect(genre)}
+          className={`${styles["genres__button"]} ${
+            selectedGenre === genre ? styles["genres__button--selected"] : null
```

Comment:
This way you'll get, literally, "null" class for not selected genres. You could fix it, for example, by specifying an empty string instead, but you could also have a look at utility tools such as [clsx](https://www.npmjs.com/package/clsx) that would allow you to construct the class string even easier and avoid using the ternary operator at all.